src/pages/member/UserNav.js

```diff
@@ -0,0 +1,112 @@
+import React, { useState } from "react";
+import {
+    Drawer,
+    List,
+    ListItem,
+    ListItemIcon
+} from "@material-ui/core";
+import { withStyles } from '@material-ui/styles';
+import DateRangeIcon from '@material-ui/icons/DateRange';
+import PersonIcon from '@material-ui/icons/Person'
+import ExitToAppIcon from '@material-ui/icons/ExitToApp';
+import { useHistory, withRouter } from "react-router-dom";
+import { Auth } from "aws-amplify";
+import { connect } from 'react-redux'
+import { logout } from '../../actions/UserActions'
+import WhiteBiztech from './images/whitebiztech.svg'
+import ColoredBiztech from './images/coloredbiztech.svg'
+import { COLOR } from '../../constants/Constants'
+
+const styles = {
+    list: {
+        width: '170px',
+        paddingTop: '95px'
+    },
+    icon: {
+        width: '49px',
+        paddingTop: '28px',
+        paddingBottom: '28px',
+        cursor: 'pointer',
+        paddingLeft: '54px'
+    },
+    iconSize: {
+        fontSize: '30'
+    },
+    logout: {
+        marginTop: '330px',
+        cursor: 'pointer',
+        paddingLeft: '48px'
+    },
+    paper: {
+        backgroundColor: COLOR.BACKGROUND_COLOR
+    }
+}
+
+function UserNav(props) {
+    const { classes } = props
+    const history = useHistory();
+
+    const selected = { color: COLOR.BIZTECH_GREEN, fontSize: '30', width: '43px' }
+    const unselected = { color: COLOR.WHITE, fontSize: '30', width: '43px' }
+    const barSelected = { borderLeft: '6px solid ' + COLOR.BIZTECH_GREEN, height: '45px', marginTop: '28px', paddingBottom: '28px', backgroundPositionX: 'left' }
+    const barUnselected = { borderLeft: '6px solid ' + COLOR.BACKGROUND_COLOR, height: '45px', marginTop: '28px', paddingBottom: '28px' }
+
+    const pathname = window.location.pathname
+    const [selectedItem, setSelectedItem] = useState(pathname)
+
+    const logout = () => {
+        Auth.signOut()
+            .then(() => {
+                props.logout()
+            })
+            .catch(err => console.log(err));
+    }
+
+    const handleEventsClick = () => {
+        history.push('/events')
+        setSelectedItem('/events')
```

Comment:
thoughts on grabbing selected item from the url path? Does this break when the page is refreshed?

Reply:
isnt it faster and just as accurate to setSelectedItem to '/events' (hard coded value) right after pushing '/events' instead of grabbing it from the URL? Wouldn't the only case where it would be wrong if '/events' failed to load or failed to navigate to that path in the url?

seems to work fine when i refresh

Reply:
oh yeah, i didnt notice this part 
```
    const pathname = window.location.pathname
    const [selectedItem, setSelectedItem] = useState(pathname)
```